src/cluster/autoscale.go

```diff
@@ -445,3 +449,23 @@ func getImageVersion(clusterID uint, cluster interface{}) map[string]string {
 
 	return selectedImageVersion
 }
+
+func setArchNodeSelector(cluster CommonCluster) map[string]string {
+	k8sVersion, err := getK8sVersion(cluster)
+	if err != nil {
+		log.Error(errors.WrapIfWithDetails(err, "unable to retrieve K8s version of cluster", "clusterID", cluster.GetID()))
+
+		return nil
+	}
+
+	compareVersion, err := semver.NewVersion("1.20.0")
+	if err != nil {
+		return nil
+	}
+
+	if k8sVersion.LessThan(compareVersion) {
```

Comment:
1.20 should be added here for this to work: https://github.com/banzaicloud/pipeline/blob/master/internal/cmd/config.go#L716